homeassistant/components/logbook/__init__.py

```diff
@@ -674,12 +676,13 @@ def augment(self, data: dict[str, Any], row: Row, context_id: str) -> None:
 
         # State change
         if context_entity_id := context_row.entity_id:
+            data[CONTEXT_STATE] = context_row.state
             data[CONTEXT_ENTITY_ID] = context_entity_id
             if self.include_entity_name:
                 data[CONTEXT_ENTITY_ID_NAME] = self.entity_name_cache.get(
                     context_entity_id, context_row
                 )
-            data[CONTEXT_EVENT_TYPE] = event_type
+            data[CONTEXT_EVENT_TYPE] = EVENT_STATE_CHANGED
```

Comment:
We now replace `None` with `state_changed` for a lot of states. Is that necessary?

Reply:
```
    if (item.context_event_type === "state_changed") {
      return "";
    }
```

So we can drop context_event_type for states and only check for CONTEXT_STATE being present.